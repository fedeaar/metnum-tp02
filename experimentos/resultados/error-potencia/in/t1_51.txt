-6.541323608289619642e+02 1.000873767026087648e+02 6.533420766517842537e+01 -2.328945563928297346e+01 -1.571483200256797019e+01 4.750461003463229304e+01 1.205754900911841787e+02 1.038886104351402473e+01 2.722472560296789368e+01 8.273430130666525883e+01 1.981860325233046183e+01 4.809163821713524101e+01 5.099808629803325033e+01 -4.367288877646457124e+00 7.881066234936882609e+01 9.758737423354313023e+01 1.210656726298285548e+02 5.841194590920022911e+00 1.369845138273150731e+01 1.020612934392624709e+02
1.000873767026087648e+02 -6.864146224372643701e+02 8.906046452964818627e+01 -1.753746230685295870e+01 -9.504669548005612256e+00 6.856749291427091464e+01 1.580248757244991964e+02 1.980590407542567277e+01 4.364809219436955345e+01 1.097073333518290639e+02 2.623250554210493490e+01 6.927398761218347545e+01 6.815901322749382985e+01 7.314901191478586640e-01 1.058259261056916500e+02 1.313386305237238787e+02 1.596095068320967982e+02 1.025899709877070443e+01 1.862304333353244701e+01 1.381863975642508251e+02
6.533420766517841116e+01 8.906046452964818627e+01 -1.798866470458416416e+02 -1.034734578929279394e+02 -8.455234793494015832e+01 3.601557586729825289e+00 1.104310191550631686e+02 -2.964437849327308783e+01 -2.546780872091503412e+01 6.769743573776977996e+01 1.651616971808613599e+01 4.536823604049819103e+00 3.834788091685122424e+01 -4.483429965934287509e+01 5.612788192457802694e+01 6.760530488193536769e+01 1.049197584496274089e+02 -1.112009168026975914e+01 8.307606104109902745e+00 6.547632489027581926e+01
-2.328945563928299123e+01 -1.753746230685296226e+01 -1.034734578929279394e+02 5.307594564737538576e+02 -3.215834488531828583e+02 -1.860494733427974836e+02 -6.697360977610420818e+00 -1.805540676732958616e+02 -2.331616643410868903e+02 -4.171359484814686880e+01 -8.615432485807264840e+00 -1.842545731057027751e+02 -4.125345015072243626e+01 -1.874340011924026044e+02 -7.815375488646800761e+01 -1.054838865467997522e+02 -3.411789154583459549e+01 -7.602044423047877331e+01 -2.024045560839519098e+01 -1.343495681136328415e+02
-1.571483200256797375e+01 -9.504669548005614033e+00 -8.455234793494015832e+01 -3.215834488531828583e+02 5.803607509150156147e+02 -1.558219198275866120e+02 8.350573915905687983e-01 -1.531555151753727273e+02 -1.970314132210983189e+02 -3.102835900060297547e+01 -6.260474864699840936e+00 -1.542619655674382102e+02 -3.235718482996809087e+01 -1.598129059336073396e+02 -6.226522533031552342e+01 -8.451536450753749818e+01 -2.248344992782881491e+01 -6.440502650818039854e+01 -1.648950286805909826e+01 -1.088482550215707079e+02
4.750461003463229304e+01 6.856749291427088622e+01 3.601557586729828841e+00 -1.860494733427974836e+02 -1.558219198275866120e+02 9.405850342238349526e+01 8.905709521286760832e+01 -7.281946667089061975e+01 -8.349440929510664944e+01 4.450692195121550299e+01 1.127720293469949020e+01 -4.569058570996653401e+01 2.048395153911589617e+01 -8.716190614125950731e+01 2.521400174913040004e+01 2.732584368367854566e+01 7.716779957928908118e+01 -2.953986817813936128e+01 1.405661004541502113e+00 1.783157321129117534e+01
1.205754900911841503e+02 1.580248757244991964e+02 1.104310191550631686e+02 -6.697360977610431476e+00 8.350573915905767919e-01 8.905709521286762254e+01 -7.187681200078144457e+02 3.053589514629303636e+01 6.104416201541248199e+01 1.327787860489843865e+02 3.169764436165566934e+01 8.983387187609652358e+01 8.307478994127673388e+01 7.986344615177704220e+00 1.295197173749020862e+02 1.610666391335341814e+02 1.921672814814162678e+02 1.515281241923572075e+01 2.303876349429699744e+01 1.703509172813147927e+02
1.038886104351401940e+01 1.980590407542567277e+01 -2.964437849327308783e+01 -1.805540676732958616e+02 -1.531555151753727273e+02 -7.281946667089061975e+01 3.053589514629303636e+01 4.383694838073703863e+02 -1.002716757069358664e+02 3.780869684922440577e+00 1.542655377223983271e+00 -7.182535441240872842e+01 -4.858999066375686127e+00 -8.792952486997425865e+01 -1.417167630119330113e+01 -2.157750433939628110e+01 1.798726229371764163e+01 -3.353941853146889684e+01 -5.542549614564733496e+00 -3.368344334177210442e+01
2.722472560296787947e+01 4.364809219436954635e+01 -2.546780872091503056e+01 -2.331616643410869187e+02 -1.970314132210983189e+02 -8.349440929510664944e+01 6.104416201541251041e+01 -1.002716757069358664e+02 2.589532301397751439e+02 2.012160285241419189e+01 5.627367230482901661e+00 -8.212563618028518420e+01 3.290584797597022160e+00 -1.122570357443681957e+02 -3.359664532439846862e+00 -9.267259193984497045e+00 4.522999788034041302e+01 -4.142784343797470825e+01 -4.486877685902603830e+00 -2.378219420892249047e+01
8.273430130666525883e+01 1.097073333518290639e+02 6.769743573776977996e+01 -4.171359484814685459e+01 -3.102835900060296481e+01 4.450692195121550299e+01 1.327787860489844434e+02 3.780869684922433471e+00 2.012160285241419899e+01 -5.264719300329923044e+02 2.150461392648035996e+01 4.523113728599747674e+01 5.452443430862579277e+01 -1.279640875292652424e+01 8.364723448935869499e+01 1.032055882378707992e+02 1.321527792360923286e+02 3.210915783986613370e+00 1.425584580043677363e+01 1.069144216759491997e+02
1.981860325233046183e+01 2.623250554210494201e+01 1.651616971808613599e+01 -8.615432485807264840e+00 -6.260474864699844488e+00 1.127720293469949375e+01 3.169764436165566934e+01 1.542655377223985047e+00 5.627367230482906990e+00 2.150461392648035996e+01 -6.488395816028140644e+02 1.144359209084697149e+01 1.315299613678296886e+01 -2.387376432955693684e+00 2.023108028992510654e+01 2.499363967859914482e+01 3.164562171228015686e+01 1.035730192986679121e+00 3.472506614980918727e+00 2.598080526513152222e+01
4.809163821713524811e+01 6.927398761218347545e+01 4.536823604049833314e+00 -1.842545731057028036e+02 -1.542619655674381818e+02 -4.569058570996653401e+01 8.983387187609652358e+01 -7.182535441240872842e+01 -8.212563618028516998e+01 4.523113728599748384e+01 1.144359209084696793e+01 8.855469219957997495e+01 2.101042525053166443e+01 -8.622319087791423442e+01 2.610179097704376261e+01 2.847002029913825538e+01 7.808885575424420722e+01 -2.911227595015064651e+01 1.594262377150947874e+00 1.915199855437765208e+01
5.099808629803325033e+01 6.815901322749382985e+01 3.834788091685121714e+01 -4.125345015072243626e+01 -3.235718482996810508e+01 2.048395153911589261e+01 8.307478994127674810e+01 -4.858999066375687015e+00 3.290584797597027489e+00 5.452443430862579277e+01 1.315299613678296886e+01 2.101042525053166443e+01 -4.394283397185913032e+02 -1.553986298256426224e+01 4.937350089187857094e+01 6.055523440916815758e+01 8.158351192510657768e+01 -1.029600166345831402e+00 8.137417368745975210e+00 6.172697790810754270e+01
-4.367288877646457124e+00 7.314901191478515585e-01 -4.483429965934287509e+01 -1.874340011924026044e+02 -1.598129059336073396e+02 -8.716190614125952152e+01 7.986344615177696227e+00 -8.792952486997425865e+01 -1.122570357443681957e+02 -1.279640875292652424e+01 -2.387376432955693240e+00 -8.622319087791423442e+01 -1.553986298256426579e+01 6.683019637715243562e+02 -3.110611815711779116e+01 -4.280580025608701078e+01 -5.469186306699257649e+00 -3.688453219359585233e+01 -8.684092870607438286e+00 -5.660008254477504863e+01
7.881066234936884030e+01 1.058259261056916500e+02 5.612788192457802694e+01 -7.815375488646802182e+01 -6.226522533031552342e+01 2.521400174913041781e+01 1.295197173749020862e+02 -1.417167630119329047e+01 -3.359664532439837981e+00 8.364723448935870920e+01 2.023108028992510299e+01 2.610179097704374840e+01 4.937350089187858515e+01 -3.110611815711779116e+01 -3.257269656116983469e+02 9.074275828533650667e+01 1.261907434014835303e+02 -4.379472773276291697e+00 1.197290873673841105e+01 9.152077615869289673e+01
9.758737423354313023e+01 1.313386305237238787e+02 6.760530488193539611e+01 -1.054838865467997238e+02 -8.451536450753749818e+01 2.732584368367853855e+01 1.610666391335341530e+02 -2.157750433939628110e+01 -9.267259193984473953e+00 1.032055882378707992e+02 2.499363967859914126e+01 2.847002029913825538e+01 6.055523440916816469e+01 -4.280580025608701078e+01 9.074275828533647825e+01 -2.573535354444546215e+02 1.563250073419386297e+02 -7.109198177190540591e+00 1.446117016050174087e+01 1.109852701610454346e+02
1.210656726298285690e+02 1.596095068320968267e+02 1.049197584496273947e+02 -3.411789154583456707e+01 -2.248344992782878649e+01 7.716779957928909539e+01 1.921672814814162678e+02 1.798726229371764163e+01 4.522999788034042012e+01 1.321527792360923286e+02 3.164562171228015686e+01 7.808885575424420722e+01 8.158351192510657768e+01 -5.469186306699250544e+00 1.261907434014835303e+02 1.563250073419386581e+02 -5.528338363002014830e+02 9.910850756807585427e+00 2.198669250058938474e+01 1.636826972754432745e+02
5.841194590920017582e+00 1.025899709877070443e+01 -1.112009168026975914e+01 -7.602044423047877331e+01 -6.440502650818039854e+01 -2.953986817813935772e+01 1.515281241923572431e+01 -3.353941853146889684e+01 -4.142784343797470825e+01 3.210915783986608041e+00 1.035730192986677345e+00 -2.911227595015064651e+01 -1.029600166345840284e+00 -3.688453219359585944e+01 -4.379472773276291697e+00 -7.109198177190544143e+00 9.910850756807583650e+00 4.570787154347188448e+02 -2.049907423500725301e+00 -1.208747257998337155e+01
1.369845138273150731e+01 1.862304333353244701e+01 8.307606104109900969e+00 -2.024045560839518743e+01 -1.648950286805909826e+01 1.405661004541501669e+00 2.303876349429699744e+01 -5.542549614564733496e+00 -4.486877685902607382e+00 1.425584580043677008e+01 3.472506614980918727e+00 1.594262377150949650e+00 8.137417368745975210e+00 -8.684092870607436510e+00 1.197290873673841105e+01 1.446117016050174087e+01 2.198669250058938474e+01 -2.049907423500725745e+00 -2.441973290872292637e+02 1.411909624035065924e+01
1.020612934392624709e+02 1.381863975642508535e+02 6.547632489027581926e+01 -1.343495681136328699e+02 -1.088482550215707079e+02 1.783157321129118245e+01 1.703509172813148211e+02 -3.368344334177210442e+01 -2.378219420892249047e+01 1.069144216759491712e+02 2.598080526513152222e+01 1.915199855437765208e+01 6.172697790810754270e+01 -5.660008254477504863e+01 9.152077615869289673e+01 1.109852701610454631e+02 1.636826972754433029e+02 -1.208747257998336622e+01 1.411909624035066102e+01 -1.823835280462581636e+02
