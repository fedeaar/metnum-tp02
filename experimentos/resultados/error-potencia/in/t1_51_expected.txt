-9.080000000000000000e+02
9.070000000000000000e+02
8.550000000000000000e+02
-8.180000000000000000e+02
7.610000000000000000e+02
-7.460000000000000000e+02
-7.300000000000000000e+02
-6.540000000000000000e+02
-6.160000000000000000e+02
5.190000000000000000e+02
-4.720000000000000000e+02
4.710000000000000000e+02
-4.000000000000000000e+02
3.810000000000000000e+02
-3.680000000000000000e+02
-2.920000000000000000e+02
-2.460000000000000000e+02
-2.180000000000000000e+02
1.410000000000000000e+02
1.330000000000000000e+02
